.github/create_opm_index.sh

```diff
@@ -9,7 +9,8 @@ echo "${INDEX_IMAGE_TAG}"
 echo "${BUNDLE_IMAGE}"
 
 echo "opm index add --bundles ${REGISTRY}/${BUNDLE_IMAGE}:${GITHUB_SHA} --tag ${REGISTRY}/${INDEX_IMAGE}:${GITHUB_SHA} -u podman --pull-tool podman"
-opm index add --bundles "${REGISTRY}/${BUNDLE_IMAGE}:${GITHUB_SHA}" --tag "${REGISTRY}/${INDEX_IMAGE}:${GITHUB_SHA}" -u podman --pull-tool podman
+#opm index add --bundles "${REGISTRY}/${BUNDLE_IMAGE}:${GITHUB_SHA}" --tag "${REGISTRY}/${INDEX_IMAGE}:${GITHUB_SHA}" -u podman --pull-tool podman
+opm index add --bundles "${REGISTRY}/${BUNDLE_IMAGE}:${GITHUB_SHA}",quay.io/openstack-k8s-operators/manila-operator-bundle:latest --tag "${REGISTRY}/${INDEX_IMAGE}:${GITHUB_SHA}" -u podman --pull-tool podman
```

Comment:
The hard coded path to quay.io/openstack-k8s-operators/manila-operator-bundle:latest would need to be fixed in a follow up. Ideally this would point to the same image we have in the go.mod file for manila dependency pinning to work correctly